frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -56,9 +62,21 @@ const TranslationPage = () => {
   >(new Map());
   const [numTranslatedLines, setNumTranslatedLines] = useState(0);
 
+  const [
+    versionHistoryStack,
+    updateVersionHistoryStack,
```

Comment:
Out of curiosity - is there a "good practice" around calling this `setVersionHistoryStack`? or doesn't really matter

Reply:
Yeah, I named this thinking more in terms of how it only add/removes one thing at a time but it should probably be named `setVersionHistoryStack` based on what it actually does